src/system.c

```diff
@@ -1006,7 +1004,6 @@ static struct optInfo options[] = {
   { 'L', "", storeString, &SyRestoring, 1}, /* must be handled in kernel  */
   { 'M', "", toggle, &SyUseModule, 0}, /* must be handled in kernel */
   { 'R', "", unsetString, &SyRestoring, 0}, /* kernel */
-  { 'a', "", storeMemory, &preAllocAmount, 1 }, /* kernel -- is this still useful */
```

Comment:
Please also remove the documentation for this in `doc/ref/run.xml`

Reply:
Also, `INSTALL.md` mentions `-a`.

Reply:
Woops, should have searched harder. Will fix!

Reply:
That install.md comment is just wrong (in my opinion), the run.xml does at least give me some idea why this option originally existed, but I checked the main OSes we run on and none of them seem to require -a anymore.